features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/data/ServiceCatalog.js

```diff
@@ -98,7 +98,7 @@ class ServiceCatalog {
             // return client.apis['Services'].searchServices();
             return MockResponses.searchServices();
         });
-        return promisedServices.then(response => response.body);
+        return promisedServices.then(response => response);
```

Comment:
this last `then` callback has no effect at all 😃  , I mean this is equal passing just `promisedServices` without attaching that `then` callback